### ISSUE TITLE
Separate big radar and small radar

### ISSUE THREAD

**Issue by lott10** [KC3Kai/KC3Kai#166]

**Comment by LostLogia4** [comment]
Where would you want the category to be separated anyway? Strategy room?

**Comment by Crazyteitoku** [comment]
I personally do not see there is any merits with this based on two reasons:

1) The list of radars isn't as big as the amount of main guns.

2) There are enough buttons already, even if one starts segregating these what is next? segregate scout seaplanes from seaplane bombers?

**Comment by LostLogia4** [comment]
How about adding tabs instead? Adding icon buttons would be a bit much, and they're all in the same category anyway.

**Comment by Crazyteitoku** [comment]
@LostLogia4 It is up to @dragonjet to decide on where to head with the need to segregate these really. Again, if needed be there is always an option to just simply Ctrl+F and type the desired characters/words/letters/numbers to find the desire equipment. Any additions to the strategy page may involve reworking the interface which is probably unnecessary for the time being.

**Comment by bamboo3250** [comment]
How about showing a list of compatible ship types for the equipment? Here we have a lot of wasted space.

![untitled](https://cloud.githubusercontent.com/assets/8078323/8447864/bc7db414-1ff1-11e5-9ecc-61102e258e7c.png)

By doing that, we can see the difference at the first glance. Later if we have sorting feature for the equipment, we also can separate them by sorting by the compatibility.

**Comment by Crazyteitoku** [comment]
@bamboo3250 I was thinking of utilising some of those spaces as for doing tallies for each equipment you have. E.g. 
Total = 25
★+3 = 1
★+9 = 3

At least I can see this sort of work of tallying the equips along with a breakdown for each of the improvements would be handy for "wikians".

Though really what equipments can be equipped to what ship is really for one to read up on. :wink: It is doable otherwise but really, that would potentially have more of upkeep as time progresses on.

**Comment by dragonjet** [comment]
hmm those new ideas are nice, adding compatible ship-types under the  basic info, and adding tally counts.. but the main post about segregating large and small radars further is going to be a `wontfix`. We currently use in-game classification of equipment types, direct from game data and is actually an auto-generated list.

It's the same as saying to further separate recon seaplane vs observer vs bomber. These are the things people memorize by heart. It can only happen if devs to change it in-game.

**Comment by lott10** [comment]
I was thinking of a simple line separating both 
like:
big radar
big radar
big radar
big radar
================================================
small radar
small radar

**Comment by lott10** [comment]
it seems that the current ordering is based on acquisition order
that can be changed right?

**Comment by bamboo3250** [comment]
@lott10 Yes, it can. I'm not against your idea. However, the game data doesn't provide enough information about the size of the equipment. That means we have to manually add more information to the database and maintain it whenever a new update comes out. As @dragonjet said, only after the game dev makes changes to the game data, will this feature be added.
